### PR TITLE
Community Highlights, 7th edition

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

7th installment of the Community Highlights.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_posts/2020-11-30-community-highlights.markdown [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+things popped up around our community, we thought was worth sharing.
+
+Troy Hunt has jumped into our wonderful world and shared his journey
+in a blog post series. Some nice examples on a DIY solution for mounting a
+wall tablet and how to arm your alarm with a toothbrush!
+
+## Troy Hunt is unraveling IoT using Home Assistant
+
+If you have never heard of Troy Hunt before, he is a web security specialist
+and mostly for his ["Have I Been Pwned"][hibp] creation.
```
Comment (balloob) [inline source/_posts/2020-11-30-community-highlights.markdown]:
```suggestion
and mostly known for creating ["Have I Been Pwned"][hibp] ([Home Assistant integration](/integration/haveibeenpwned/).
```

File: source/_posts/2020-11-30-community-highlights.markdown [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+Troy has been diving into the world of IoT and started automating his home,
+using... Home Assistant! In 5 blogs, he writes about his journey in the big
+world of IoT:
```
Comment (balloob) [inline source/_posts/2020-11-30-community-highlights.markdown]:
```suggestion
Troy has been diving into the world of IoT and started automating his home,
using... Home Assistant!

> _"I'm massively impressed with HA and nothing else I've seen along my IoT journey comes even close to comparing. If you're going to do IoT in any meaningful way, you start with HA."_ – Troy Hunt

In 5 blogs, he writes about his journey in the big
world of IoT:
```

**Review by balloob — DISMISSED** [review]

File: source/_posts/2020-11-30-community-highlights.markdown [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+## Troy Hunt is unraveling IoT using Home Assistant
+
+If you have never heard of Troy Hunt before, he is a web security specialist
+and mostly known for creating ["Have I Been Pwned"][hibp] ([Home Assistant integration](/integration/haveibeenpwned/)).
+
+Troy has been diving into the world of IoT and started automating his home,
+using... Home Assistant!
+
+> _"I'm massively impressed with HA and nothing else I've seen along my IoT journey comes even close to comparing. If you're going to do IoT in any meaningful way, you start with HA."_ – Troy Hunt
```
Comment (balloob) [inline source/_posts/2020-11-30-community-highlights.markdown]: Forgot our blockquote automatically adds quotes

image

```suggestion

> I'm massively impressed with HA and nothing else I've seen along my IoT journey comes even close to comparing. If you're going to do IoT in any meaningful way, you start with HA.
```

File: source/_posts/2020-11-30-community-highlights.markdown [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+<p lang="en" dir="ltr">Have you ever seen a house alarm system controlled by a toothbrush? Here our <a href="https://twitter.com/VanderbiltInd?ref_src=twsrc%5Etfw">@VanderbiltInd</a> alarm is integrated into <a href="https://twitter.com/home_assistant?ref_src=twsrc%5Etfw">@home_assistant</a> . Our toothbrush is ble enabled, it’s state is read by an ESP32 chip and sent into <a href="https://twitter.com/home_assistant?ref_src=twsrc%5Etfw">@home_assistant</a> <a href="https://t.co/T9XGi1Zjmb">pic.twitter.com/T9XGi1Zjmb</a></p>&mdash; Ian (@imduffy15) <a href="https://twitter.com/imduffy15/status/1329569523855478784?ref_src=twsrc%5Etfw">November 19, 2020</a>
+</blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## Home Assistant Conference
+
+This isn't a highlight yet, but without a doubt, it is definitely is going to
+become one of the highlights of this year: The Home Assistant Conference.
+
+<a href="/conference"><img src='/images/conference/conference-header.png' alt='Home Assistant Conference header' class='no-shadow'></a>
```
Comment (balloob) [inline source/_posts/2020-11-30-community-highlights.markdown]: Can we embed a screenshot of the schedule instead

Comment (frenck) [reply source/_posts/2020-11-30-community-highlights.markdown]: Yes, we can now 👍  (the screenshot was published after this PR).

Will adjust.

File: source/_posts/2020-11-30-community-highlights.markdown [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+Sunday, 13 December, we'll be looking back at last year; but more importantly,
+lots of great talks are planned by speakers from our community about all
+different kinds of subjects:
+
+<a href="/conference"><img src='/images/conference/schedule.png' alt='Home Assistant Conference Schedule' class='no-shadow'></a>
+
+Above all, some new and exciting stuff is going to be announced as well! So,
+ensure you don't miss it.
+
+[Check out our Home Assistant Conference page on how to join!](/conference)
```
Comment (balloob) [inline source/_posts/2020-11-30-community-highlights.markdown]: What about something like:

> Get your $1 ticket or check the Home Assistant Conference page for other ways on how to watch.

Comment (frenck) [reply source/_posts/2020-11-30-community-highlights.markdown]: Updated

**Review by balloob — APPROVED** [review]

**Pull request merged**